quizblock/templates/quizblock/quiz_detail.html

```diff
@@ -15,8 +15,8 @@
             padding-left: 1.5em;
             background-color: #ddd;
         }
-        #questions li span { position: absolute; margin-left: -1.3em; }
-        .dragging {border: #f60 solid 1px;}
+        #questions li .draghandle { cursor: move; }
```

Comment:
ah, right, my mistake. The "view" is quizblock.html and then the edit detail view is "quiz_detail" per CBV conventions.